### ISSUE TITLE
Typescript definition files errors

### ISSUE THREAD

**Issue by hobojoe** [chsakell/aspnet5-angular2-typescript#28]
Hi,

First of all I would like to thank you for the great tutorial on Angular2 SPA.

I've downloaded the complete solution but I can't seem to build it because I am facing TS definition files errors.

`Property 'map' does not exist on type 'Observable<Response>'`
![image](https://cloud.githubusercontent.com/assets/1886514/20458406/a430e6ae-ae8b-11e6-85c3-4b9f8b3f54b5.png)

It keeps complaining on the angular definition files as if they couldn't be found.

I have followed all the installation steps described on the blog, I have even reinstalled VS but still no luck.

Am I missing something?

I have Visual Studio 2015 Community.

I have also tried on VS Code, same problem.

Can anyone assist me? Thanks

**Comment by chsakell** [comment]
Make sure you follow the instructions on the [README.md](https://github.com/chsakell/aspnet5-angular2-typescript/blob/master/README.md) file. For start, it seems that you haven't downloaded the npm packages.

**Comment by hobojoe** [comment]
Hi,

I followed the instructions and installed everything. The npm packages are installed. If I check the node_modules folder has all the needed files I even managed to find the one which declares de MAP function missing on my errors but for some reason VS does not recognize them.

I tried using your complete code and also by following the blog post and creating my own solution. The issue is the same.

I am not sure what could be the problem.

Any other ideas?

Thanks

**Comment by chsakell** [comment]
Have you installed the latest [TypeScript](https://www.microsoft.com/en-us/download/details.aspx?id=48593) version for VS 2015? This might help you.

**Comment by hobojoe** [comment]
Hi, I believe I have 2.0.10 installed.

When I get home I'll double check and let you know.

**Comment by hobojoe** [comment]
Hi,

I do have the TS version 2.0.10 installed.

After installing the Typescript for VS2015 your solution started working, the errors are gone.

I then went to the project I was creating by following you blog post but unfortunately the errors persist on it. I've checked everything and it seems correct.

So in order to double check I started a brand new project, followed your blog post up until the angular part where I only added the data.service.ts file to check if it would work. It didn't - same errors.

I have added this new solution to github:
https://github.com/hobojoe/TestApp

If you could check and see if I am missing something it would be great.

I am looking forward to start using your "template" on my new app and this TS thing is killing me!

Thanks in advance!

**Comment by hobojoe** [comment]
Any thoughts on what might be the issue?

Thanks

**Comment by marcelo-maciel** [comment]
@hobojoe i installed a new machine with win10 and had many errors, including few  map errors.
I solved my issues running again all global npm commands and after instaling 'typings' again, in same folder of appsettings.json.

I also installed these apps:
DotNetCore.1.0.1-SDK.1.0.0.Preview2-003133-x64.exe
DotNetCore.1.0.1-VS2015Tools.Preview2.0.3.exe
dotnet-dev-win-x64.1.0.0-preview2-1-003177.exe
And all VS2015 updates.
